conjur/variable/variable_logic.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+"""
+VariableLogic module
+
+This module is the business logic for executing the VARIABLE command
+"""
+
+# Builtins
+import json
+import logging
+
+# pylint: disable=too-few-public-methods
+class VariableLogic:
+    """
+    VariableLogic
+
+    This class holds the business logic for executing and manipulating
+    returned data
+    """
+    def __init__(self, client):
+        self.client = client
+
+    # pylint: disable=logging-fstring-interpolation
+    def get_variable(self, variable_data):
+        """
+        Method to handle all get action activity
+        """
+        logging.debug(f"{variable_data}")
```

Comment:
cant we just do
logging.debug(variable_data) ?

Reply:
yes